src/content/protocols/chainlink-oracles.md

```diff
@@ -0,0 +1,112 @@
+x: "https://x.com/chainlink"
+github: ["https://github.com/smartcontractkit"]
+defillama_slug: [""]
+chain: "Ethereum"
+type: "Oracles"
+logo: "/images/infrastructure-logos/chainlink-logo.png"
+stage: "I0"
+protocols: ["Morpho", "Aave v3", "Compound-V3", "Liquity"]
+reasons: []
+risks: ["L", "H", "L", "M", "L"]
+author: ["mmilien_"]
+submission_date: "2025-05-20"
+publish_date: "2025-05-20"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Chainlink offers real-time price feeds for several assets, which are broadly used in DeFi.
+
+Chainlink's price feeds aggregate several data sources and publish them onchain using a combination of the Decentralized Data Model and Offchain Reporting. More information on those concepts can be found directly [in their docs](https://docs.chain.link/data-feeds#price-feeds).
+
+This report is a limited assessment of Chainlink's price feeds to assess the centralization of DeFi projects that rely on Chainlink for prices. We therefore limited our investigation to the main centralization vectors that we have found and listed below.
+
+# Protocol Analysis
+
+An overview of the Chainlink Price Feeds can be seen below.
+
+![Overview of Chainlink price feeds](./diagrams/chainlink-oracles-overview.png)
+
+The core of the protocol is the aggregator contract (`AccessControlledOCR2Aggregator`) and its proxy (`EACAggregatorProxy`), the _Node Operators_, and the [Chainlink multisig](#security-council).
+
+The Chainlink _Node Operators_ are a set of well-established companies in diverse economic sectors. The _Node Operators_ reach a consensus by collaborating offchain using several data sources. Once enough _Node Operators_ agree on a price, one transmitter can send the new price and a sufficient amount of signatures to the `AccessControlledOCR2Aggregator` contract. The entry point for users is the `EACAggregatorProxy`, which will in turn query the `AccessControlledOCR2Aggregator`. Both contracts are under the control of the [Chainlink multisig](#security-council) multisig, which could effectively update which Aggregator is used by the Proxy. This could arbitrarily change the entire logic of the price feed.
+
+# Rating
+
+As mentioned in the [protocol analysis](#protocol-analysis), the [Chainlink multisig](#security-council) has permissions to change the price feeds at any given time without delay. This could change the entire logic of the price feeds, disabling further updates or rendering inaccurate data.
+
+Those changes can be made **without delay** and the [Chainlink multisig](#security-council) does not follow the requirements for a _security council_.
+
+## Conclusion
+
+Chainlink's protocol exposes centralized permissions on the price feeds, which leads to a _High Upgradeability_ risk score. Those permissions are not protected by an _Exit Window_ nor a _Security Council_. We believe that this brings the Chainlinik price feeds to a **High Centralization**, equivalent to _Stage 0_, as a dependency.
```

Comment:
Chainlink's oracle protocol exposes critical permissions that are not protected by an _Exit Window_ nor a _Security Council_. This results in a **High Centralization** score.